packages/client/src/client/App/Credit/CreditRfqs/CreditRfqConfirmation.tsx

```diff
@@ -97,13 +110,15 @@ export const CreditRfqCreatedConfirmation = () => {
   const { direction, dealerIds, quantity, instrument } = confirmation.request
 
   return confirmation ? (
-    <ConfirmationPill direction={direction}>
-      You have sent an {direction} RFQ for {formatter(quantity)}{" "}
-      {instrument?.name} to {dealerIds.length} dealers
-      <IconWrapper direction={direction} onClick={onDismissMessage}>
-        <FaTimes />
-      </IconWrapper>
-    </ConfirmationPill>
+    <ConfirmationPillWrapper>
+      <CreatedRFQConfirmationPill direction={direction}>
+        You have sent an {direction} RFQ for {formatter(quantity)}{" "}
```

Comment:
let's correct this while we are here .. right now, it gives us
```You have sent an Buy/Sell RFQ ...```

Reply:
see other util somewhere as I pestered Tammy to centralise the handling of the RFQ quantity (always in 000s, so need to format appropriately everywhere we present quantity to user (after they have entered the quantity in the smaller form, ofc)

Reply:
see below, there is no other util .. must not have been created 🤦‍♀️

Reply:
this bit goes away, anyhow, right?